applications/StructuralMechanicsApplication/tests/response_function_tests/adjoint_max_stress_response_parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "cantilever_beam",
+        "model_part_name" : "cantilever_beam",
+        "domain_size"     : 3,
+        "time_step"       : 1.0,
+        "start_time"      : 0.0,
+        "end_time"        : 1.0,
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0
+    },
+    "kratos_response_settings":{
+        "response_type"        : "adjoint_max_stress",
+        "gradient_mode"        : "semi_analytic",
+        "step_size"            : 1.0e-6,
+        "critical_part_name"   : "stress_region",
+        "stress_type"          : "VON_MISES_STRESS",
```

Comment:
style-guide: `von_mises_stress`

Reply:
This follows thee existing code, see comment below.